### PR TITLE
Remove chart for analog polls with a majority & fix random TypeError

### PR DESCRIPTION
**Pull request by jsangmeister**

### CODE REVIEW

**Comment by jsangmeister** [comment]
It seems like a lot was reformatted by `npm run cleanup`. Is this an error on my part? What is the preffered way to clean up client code nowadays? (@tsiegleauq @FinnStutzenstein)

**Comment by jsangmeister** [comment]
Changes are in `motion-poll-detail-content.component.html`, `base-poll-detail.component.ts` and `view-base-poll.ts` (if I didn't miss anything)

**Comment by tsiegleauq** [comment]
npm run cleanup was still the way to go. Perhaps your prettier is outdated, try deleting your node_modules and run `npm install`

**Comment by tsiegleauq** [comment]
I cleaned it up

File: client/src/app/shared/components/motion-poll-detail-content/motion-poll-detail-content.component.html [hunk 1]
```diff
@@ -43,7 +43,7 @@
         </table>
 
         <!-- Chart -->
-        <div class="doughnut-chart">
+        <div *ngIf="!poll.hasSpecialVoteValues" class="doughnut-chart">
```
Comment (emanuelschuetze) [inline client/src/app/shared/components/motion-poll-detail-content/motion-poll-detail-content.component.html:46]: This line does not work on motion poll slide (chart is always visible). - I have to merge this PR now (rest works). We can fix it later in a new PR.

**Review by emanuelschuetze — APPROVED** [review]

**Pull request merged**